ReactiveCocoaTests/Swift/SignalSpec.swift

```diff
@@ -14,10 +14,29 @@ import ReactiveCocoa
 class SignalSpec: QuickSpec {
 	override func spec() {
 		describe("init") {
-			pending("should run the generator immediately") {
-			}
-
-			pending("should keep signal alive if not terminated") {
+			let testError = NSError(domain: "SignalSpec", code: 1, userInfo: nil)
+			var testScheduler: TestScheduler!
+			
+			beforeEach {
+				testScheduler = TestScheduler()
+			}
+			
+			it("should run the generator immediately") {
+				var didRunGenerator = false
+				Signal<Int, NoError> { observer in
+					didRunGenerator = true
+					return nil
+				}
+				
+				expect(didRunGenerator).to(beTrue())
```

Comment:
Can you please use `beTruthy` and `beFalsy`, since those are more portable to other versions of Nimble?